std_msgs/Header header
string vehicle_id # name of observing vehicle
string observed_id # name of observed vehicle

ugvposewithcov ugvEst			# current estimated pose of ugv
ugvposewithcov meas_pose	# pose measured by ugv sensor
